.template/addons/bootstrap/application.js.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
-insert_into_file 'app/javascript/packs/application.js', before: %r{import 'translations/translations'.+\n} do
+insert_into_file 'app/javascript/packs/application.js', after: %r{import "channels"\n} do
```

Comment:
@hoangmirs be careful as the previous implement we would like to put `vendor/bootstrap` under `core-js` stuff

![image](https://user-images.githubusercontent.com/11751745/104284743-1ca8ca80-54e5-11eb-9798-1b4098b5efa9.png)

Reply:
For that reason, I suggest removing this change and keeping the previous implementation @hoangmirs